ui/widgets/nuxeo-html-editor.js

```diff
@@ -70,44 +72,53 @@ import './quill/quill-snow.js';
             </select>
           </span>
           <span class="ql-formats">
-            <button class="ql-bold"></button>
-            <button class="ql-italic"></button>
-            <button class="ql-underline"></button>
-            <button class="ql-strike"></button>
+            <button class="ql-bold" aria-label$="[[i18n('htmlEditor.bold')]]"></button>
```

Comment:
Should we also add a `title` to these buttons, bound to the same labels used in aria-label?

Reply:
I agree, it would give us a tooltip for free.